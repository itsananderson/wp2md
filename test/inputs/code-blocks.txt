== Code Blocks ==

`this code should stay inline`

This code `block
should be treated as inline`, even though it's on multiple lines

`
// This code is multi-line
// And should be treated as a multi-line block
`

`
<?php
// This code starts with a <?php tag
// It should be annotated as PHP code
`

`
<p>maybe some html</p>
<?php /* then some php */ ?>
`