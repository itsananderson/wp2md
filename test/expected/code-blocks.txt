Code Blocks
-----------

`this code should stay inline`

This code `block
should be treated as inline`, even though it's on multiple lines

```
// This code is multi-line
// And should be treated as a multi-line block
```

```php
<?php
// This code starts with a <?php tag
// It should be annotated as PHP code
```

```php
<p>maybe some html</p>
<?php /* then some php */ ?>
```